config/community_images.json

```diff
@@ -0,0 +1,10 @@
+[{
+    "id": "Pegasus",
+    "label": "Pegasus (Pegasuspy 1.0, Python 3.7, scPlot 0.0.16, harmony-pytorch 0.1.3)",
+    "version": "1.0",
+    "updated": "2020-07-21",
+    "packages": "https://storage.googleapis.com/terra-docker-image-documentation/placeholder.json",
+    "image": "cumulusprod/pegasus-terra:1.0",
+    "requiresSpark": false,
+    "isCommunity": true
```

Comment:
The way I wrote the UI code, we don't have to include it everywhere its false, so its an 'optional' field. this may change pending the UI PR. I'll document it in a community contributor guide once this is ironed out.